### PR TITLE
feat(llgo/c/inih): Add INIReader support

### PR DESCRIPTION
**Pull request by morpingsss**
- [ ] readme.md

### CODE REVIEW

File: c/inih/inih.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	LLGoPackage = "link: $(pkg-config --libs inih); -linih"
+	LLGoPackage = "link: $(pkg-config --libs inih) $(pkg-config --libs inihReader); -linih -linihReader -lINIReader"
```
Comment (aofei) [inline c/inih/inih.go]:
```suggestion
	LLGoPackage = "link: $(pkg-config --libs inih inihReader); -linih -linihReader -lINIReader"
```

File: c/inih/inihReader.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//go:linkname NewIniReader C.new_ini_reader
+func NewIniReader(fileName *c.Char) *IniReader
+
+//go:linkname DeleteIniReader C.delete_ini_reader
+func DeleteIniReader(reader *IniReader)
+
+// llgo:link (*IniReader).GetInteger C.ini_get_integer
+func (*IniReader) GetInteger(section *c.Char, name *c.Char, defaultValue c.Long) c.Long {
+	return 0
+}
```
Comment (aofei) [inline c/inih/inihReader.go]:
```suggestion
type Reader struct {
	Unused [0]byte
}

//go:linkname NewReader C.new_ini_reader
func NewReader(fileName *c.Char) *Reader

//go:linkname DeleteReader C.delete_ini_reader
func DeleteReader(r *Reader)

// llgo:link (*Reader).GetInteger C.ini_get_integer
func (*Reader) GetInteger(section *c.Char, name *c.Char, defaultValue c.Long) c.Long {
	return 0
}
```

File: c/inih/inihReader.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+// llgo:type C
+type Reader struct {
+	Unused [0]byte
+}
+
+//go:linkname NewReader C.new_ini_reader
+func NewReader(fileName *c.Char) *Reader
+
+//go:linkname DeleteReader C.delete_ini_reader
+func DeleteReader(r *Reader)
```
Comment (aofei) [inline c/inih/inihReader.go]:
```suggestion
// llgo:link (*Reader).Free C.delete_ini_reader
func (*Reader) Free() {}
```

**Comment by xushiwei** [comment]
Please send pr to https://github.com/goplus/llgoexamples

**Pull request closed without merging**